extensions/amp-story/1.0/test/validator-amp-story-grid-layer-presets.out

```diff
@@ -0,0 +1,54 @@
+PASS
+|  <!--
+|    Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
```suggestion
|    Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Reply:
Might want to run `gulp validator --update-tests` to get the new date automatically and verify the changes to the protoascii are reflected here.